priv/cabbage/apps/itest/lib/client.ex

```diff
@@ -96,10 +95,15 @@ defmodule Itest.Client do
     submit_typed(typed_data_signed)
   end
 
-  def get_utxos(address) do
-    payload = %AddressBodySchema1{address: address}
-    {:ok, response} = Account.account_get_utxos(WatcherInfo.new(), payload)
-    Poison.decode!(response.body, as: %{"data" => [%Utxo{}]})["data"]
+  def get_utxos(params) do
+    default_paging = %{page: 1, limit: 200}
+    %{address: address, page: page, limit: limit} = Map.merge(default_paging, params)
+
+    {:ok, response} =
+      Account.account_get_utxos(WatcherInfo.new(), %AddressBodySchema1{address: address, page: page, limit: limit})
+
+    data = Poison.decode!(response.body)
```

Comment:
where did Poison come from? I think we're using Jason everywhere?

Reply:
I think it was there since the beginning, I can change it to jason 🙆‍♂

Reply:
pls!

Reply:
done